src/lambda_function.py

```diff
@@ -79,8 +86,13 @@ def lambda_handler(event, context):
             retval = "I'm afraid I did not understand that command. Use 'jarvis help' for available commands."
             print 'Error: ' + format(str(e))
 
-    post_to_slack(retval)
-    
+    #if sendto: is in args then send jarvis message to slack channel in args
+    if sendto_data:
+        send_to_slack(retval, sendto_data[0])
+    else:
```

Comment:
Do you want to print locally what you're sending to the other channel as well? Not sure if that makes sense or not...but perhaps an option